sass/includes/_featured-article.scss

```diff
@@ -1,7 +1,10 @@
 .featured-article {
     display: flex;
-    //margin-bottom: 4rem;
-    margin-bottom: 2rem;
+    flex-direction: row;
+
+    @media only screen and (max-width: $screen__md) {
+        flex-direction: column;
+    }
```

Comment:
Note: small tidy up, we are relying on `container` which worked with the old designs but ideally we should transition to a not using a a fixed width system as it cannot scale well for larger device resolutions.